client/js/libs/handlebars/friendlysize.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+const sizes = ["Bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"];
+
+module.exports = function(size) {
+	// Loosely inspired from https://stackoverflow.com/a/18650828/1935861
+	const i = size > 0 ? Math.floor(Math.log(size) / Math.log(1024)) : 0;
+	return `${parseFloat((size / Math.pow(1024, i)).toFixed(1))} ${sizes[i]}`;
```

Comment:
I personally think this is a case where templating actually overly complicates the line, but nbd

Reply:
Fair enough, I can get behind that:

```js
`${parseFloat((size / Math.pow(1024, i)).toFixed(1))} ${sizes[i]}`;
// vs.
parseFloat((size / Math.pow(1024, i)).toFixed(1)) + " " + sizes[i];
```

I can switch to second line if @thelounge/maintainers prefer, unless we have the https://eslint.org/docs/rules/prefer-template rule. I know it's rather insignificant here, but that would apply to future changes as well.

Reply:
Wouldn't it be better to pull it out and still use a template string?

```
const fixedSize = parseFloat((size / Math.pow(1024, i)).toFixed(1));
return `${fixedSize} ${sizes[i]}`
```

I feel like just not having it as a template string doesn't make it any more readable. It's still confusing to read.

Reply:
that or @astorije's solution is better than what there is now imo